examples/plugin/test-plugin-dynamic.cpp

```diff
@@ -0,0 +1,24 @@
+//~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~//
+// Copyright (c) 2016-20, Lawrence Livermore National Security, LLC
+// and RAJA project contributors. See the RAJA/COPYRIGHT file for details.
+//
+// SPDX-License-Identifier: (BSD-3-Clause)
+//~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~//
+
+#include "RAJA/RAJA.hpp"
+#include <cstdlib>
+
+int main(int RAJA_UNUSED_ARG(argc), char **RAJA_UNUSED_ARG(argv[]))
+{
+  // Loading .so files as plugins in current directory.
+  RAJA::util::PluginRegistry::Add<RAJA::util::RuntimePluginLoader> P("RuntimePluginLoader", "RuntimePluginLoader");
```

Comment:
@JohnHynes please change 'Add' to 'add' (lowercase) so this follows the convention of other RAJA methods. Thanks.

Reply:
@rhornung67 - this change is orthogonal. The PluginRegistry was added with the uppercase method a while back.